autoload/quickrun.vim

```diff
@@ -528,6 +528,14 @@ let g:quickrun#default_config = {
 \   'command': 'sqlite3',
 \   'exec': ['%c %o < %s'],
 \ },
+\ 'purescript': {
+\   'type': executable('pulp') ? 'purescript/pulp' : '',
+\ },
+\ 'purescript/pulp': {
+\   'command': 'pulp',
+\   'cmdopt': '--monochrome',
+\   'exec': '%c %o run',
+\ },
```

Comment:
Please add the new entries in alphabetical order.

Reply:
OK, thank you for your advice.